### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- Updated names of few functions:
   `optInDaoApp` => `optInToAppUsingSecretKey`
   `optInToApp` => `optInToAppUsingLogicSig`
- Disabled Add Proposal if address is not selected
- Added `optInDaoApp` in Add Proposal.
- Added description in deposit token.
- Added minimum deposit amount from dao app global state.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "sigmadao-ui",
+	"name": "sigma-dao",
```
Comment (rmeena840) [inline package.json]:
```suggestion
	"name": "SigmaDAO",
```

File: src/pages/Proposal.vue [hunk 2]
```diff
@@ -237,6 +244,29 @@ export default defineComponent({
 		toggleModalVisible() {
 			this.isModalVisible = !this.isModalVisible;
 		},
+		async optInDaoApp() {
+			try {
+				if (this.daoStore.dao_id) {
+					const isOptIn = await isApplicationOpted(
```
Comment (rmeena840) [inline src/pages/Proposal.vue]:
```suggestion
					const isOptedIn = await isApplicationOpted(
```

File: src/pages/Proposal.vue [hunk 3]
```diff
@@ -237,6 +244,29 @@ export default defineComponent({
...
+						this.walletStore.address,
+						this.daoStore.dao_id
+					);
+					if (!isOptIn) {
```
Comment (rmeena840) [inline src/pages/Proposal.vue]:
```suggestion
					if (!isOptedIn) {
```

File: src/pages/Proposal.vue [hunk 4]
```diff
@@ -325,7 +360,7 @@ export default defineComponent({
 								sk: new Uint8Array(0),
 							},
 							toAccountAddr: getApplicationAddress(this.daoStore.dao_id),
-							amount: 15,
+							amount: globalStateMinAmount as number,
```
Comment (rmeena840) [inline src/pages/Proposal.vue:400]: Good catch!!

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed

File: src/UIKit/TablePagination.vue [hunk 5]
```diff
@@ -34,10 +34,15 @@
 import { ROWS_PER_PAGE } from "@/constants";
 import { PaginationCallType } from "@/types";
 import { defineComponent, ref } from "vue";
+import { LeftOutlined, RightOutlined } from "@ant-design/icons-vue";
```
Comment (rmeena840) [inline src/UIKit/TablePagination.vue:37]: Imported but not using them???

File: package.json (hunk 1 above)
Comment (Megha-Dev-19) [reply package.json]: There are certain provision regarding naming convention, using this throwed me the error, will see if there is any solution to it

File: src/UIKit/TablePagination.vue (hunk 5 above)
Comment (Megha-Dev-19) [reply src/UIKit/TablePagination.vue:37]: Wasn't able to see the next, prev icons in pagination, when debugged I found the issue, so fixed it
LeftOutlined -> <left-outlined/>
RightOutlined -> <right-outlined/>

File: package.json (hunk 1 above)
Comment (rmeena840) [reply package.json]: Pinged you a better approach

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**